test/TestUtilities/Test.Utility/Signing/SigningTestUtility.cs

```diff
@@ -99,7 +99,9 @@ public static class SigningTestUtility
             string subjectName,
             Action<X509V3CertificateGenerator> modifyGenerator,
             string signatureAlgorithm = "SHA256WITHRSA",
-            int publicKeyLength = 2048)
+            int publicKeyLength = 2048,
+            DateTime? notAfter = null,
+            DateTime? notBefore = null)
```

Comment:
Technically, you can discard changes in this file.  The custom generator is called later, so you can override the NotBefore and NotAfter dates like [here](https://github.com/NuGet/NuGet.Client/blob/ea7de5af727e72569e416d19bfbeba7dd162f12b/test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/TimestampProviderTests.cs#L89).